getsolus/packages
# ding: Update to 1.9

**Summary**

- Enhanced English-German dictionary (380,000 entries)
- Bugfixes
- Add `homepage` key to `package.yml` (Part of getsolus/packages#411)
- Add appstream metainfo (Part of getsolus/packages#1389)

**Test Plan**

- Run the app and use all functions (Tested on GNOME-Wayland and XFCE-X11)
- Verified the added homepage loads information about the package
- Verify metainfo with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**
